iheartradio/pyPodcastParser
# episode level data

## JIRA Ticket
-  https://ihm-it.atlassian.net/browse/IHRACP-3057

## Description
- We should verify that our parsing library can parse episodes, and add the functionality if it does not exist. Identify what can be deprecated.

#### Major Changes:
Fields added:
Interactive 

Created uts for the following fields:
Episode_number
Season
Episode_Type 
Publish_Date
Image_url
Interactive 

Fields to remove (proposed): 
tv_rating
enddate
expiration
orignal air date
image -- seems to be null. url already exists  

Existing uts for episode fields:(line 61 of test)
description
author 
duration
explicit 
order 
subtitle
summary
enclosed url 
guid
linktitle -- just the title with no spaces
date time -- start date 

others:
last_modified -- populated internally through a condition.

## How To Test

1. pyenv exec tox. 
2. Verify all uts passed and the list above is accurate.
3. Verify all additional edge case listed are handled

Related PRs:
N/A
